plugin.go

```diff
@@ -44,6 +44,9 @@ func (p Plugin) Exec() error {
 
 	var commands []*exec.Cmd
 	remote := p.Config.Remote
+	if len(p.Config.Secrets) != 0 {
+		exportSecrets(p.Config.Secrets)
```

Comment:
For other plugins I'm remapping the secret variables to the correct env variable via the environment attribute within the config